packages/sirius-web/backend/sirius-web/src/main/resources/application.properties

```diff
@@ -16,8 +16,12 @@ spring.datasource.url=jdbc:postgresql://localhost:5438/sirius-web-db
 spring.datasource.username=dbuser
 spring.datasource.password=dbpwd
 
+spring.mvc.pathmatch.matching-strategy=ant_path_matcher
```

Comment:
My IDE does not like this syntax, but accepts `ant-path-matcher` (which seems to work too).

Reply:
It's the same in IntelliJ too, I was a bit surprised by that too when we migrated to Spring Boot 3.x

Reply:
Espcially since when the property is mussing, Spring Boots says:
```
Fix this pattern in your application or switch to the legacy parser implementation with 'spring.mvc.pathmatch.matching-strategy=ant_path_matcher'.
```